src/_ert_forward_model_runner/job.py

```diff
@@ -115,7 +122,7 @@ def ensure_file_handles_closed():
         max_memory_usage = 0
         while exit_code is None:
             try:
-                memory = process.memory_info().rss + sum(
+                memory_rss = process.memory_info().rss + sum(
```

Comment:
What is rss?

Reply:
"resident set size". One of the available measures from the Linux kernel on memory usage. There are others, see the original issue.